internal/bucket/bucket.go

```diff
@@ -0,0 +1,49 @@
+package bucket
+
+import (
+	"errors"
+	"fmt"
+
+	shell "github.com/ipfs/go-ipfs-api"
+	bt "github.com/sonr-io/sonr/x/bucket/types"
+)
+
+var (
+	errContentNotFound = func(id string) error {
+		if id != "" {
+			return fmt.Errorf("could not find content with id: %s", id)
+		}
+
+		return errors.New("could not find content")
+	}
+)
+
+type Bucket interface {
```

Comment:
I feel like we should have more here but not sure why

Reply:
List content without resolving?

Reply:
thats a good idea, ill add `GetContent() []*BucketContent`